engines/sci/detection.cpp

```diff
@@ -85,11 +85,11 @@ static const PlainGameDescriptor s_sciGameTitles[] = {
 	{"lsl3",            "Leisure Suit Larry 3: Passionate Patti in Pursuit of the Pulsating Pectorals"},
 	{"mothergoose",     "Mixed-Up Mother Goose"},
 	{"pq2",             "Police Quest II: The Vengeance"},
-	{"qfg1",            "Quest for Glory I: So You Want to Be a Hero"},	// Note: There was also a SCI11 VGA remake of this (further down)
+	{"qfg1",            "Quest for Glory: So You Want to Be a Hero"},	// Note: There was also a SCI11 VGA remake of this (further down)
```

Comment:
If you are going by box art, this should be "Hero's Quest: So You Want to Be a Hero"
https://www.mobygames.com/game/heros-quest-so-you-want-to-be-a-hero/cover-art/gameCoverId,227899/